streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -144,7 +144,6 @@
     private static final long EOS_DEFAULT_COMMIT_INTERVAL_MS = 100L;
 
     public static final int DUMMY_THREAD_INDEX = 1;
```

Comment:
There are some refactor code related to KIP-695 (https://github.com/apache/kafka/commit/4d28391480fd8c547a63af119bba67fceb5d2ede#diff-07a8a87d3ee84c3e96918aa131033780cf4bc4d4a12ae376e91c059a9e02d229R142)

Reply:
Thanks, @chia7712 .

Are you pointing out that this doesn't completely revert the commits? This is true. I had to resolve a bunch of conflicts anyway, so I went over the diff and kept most of the "code cleanup" changes that I had done in those commits. That way, code in 2.8 will be as close as possible to trunk, making it easier to cherry-pick fixes in the future.